### DISCUSSION TITLE
technical terms and proper nouns with deepgram

### DISCUSSION THREAD

**Discussion by rawwerks** [deepgram/community#805] [General help] [answered]
what is the deepgram equivalent of a whisper prompt that includes a "spelling glossary"? 

```
# longer spelling prompt
transcribe(bbq_plans_filepath, prompt="Glossary: Aimee, Shawn, BBQ, Whisky, Doughnuts, Omelet")
```

(from https://cookbook.openai.com/examples/whisper_prompting_guide)

it would seem that `keywords` is the way to go, but there are no examples in the docs. https://developers.deepgram.com/docs/keywords

i can't evaluate deepgram without this ability, and i am not interested in customizing a model until i can test it head-to-head against whisper.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@rawwerks  you are correct, keywords will do this for you.   This Doc also talks about the use of Search vs. Keywords:

https://developers.deepgram.com/docs/keywords-vs-search

Here's an example of how you'd use keywords:

```curl
curl \
  --request POST \
  --header 'Authorization: Token YOUR_DEEPGRAM_API_KEY' \
  --header 'Content-Type: audio/wav' \
  --data-binary @youraudio.wav \
  --url 'https://api.deepgram.com/v1/listen?keywords=KEYWORD:INTENSIFIER'
```

It accepts multiple instances in the query string  like `keywords=medicine&keywords=prescription:2`. as well as an intensifier value.

What you are doing here is describing the Keywords to which the model should pay particular attention to boosting or suppressing to help it understand context. 

Intensifier indicates how much you want to boost it. The default Intensifier is one (1). An Intensifier of two (2) equates to two boosts multiplied in a row, whereas zero (0) is equivalent to not specifying a keywords parameter at all.

Our [API Playground](https://playground.deepgram.com/?endpoint=listen&keywords=test%3A1&language=en&model=nova-2) is a great place to test this feature out as well.
